Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/expectations.js

```diff
@@ -88,10 +89,11 @@ Expectations.stateToIdMap = {
     WARNING: 0x38,
     PASS: 0x40,
 };
-Expectations.failureTypes = ['warning', 'failed', 'timedout', 'crashed'];
+Expectations.failureTypes = ['warning', 'failed', 'image', 'timedout', 'crashed'];
```

Comment:
I think we need to update the legend in `Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/timeline.js` too

Reply:
Got it. I will work on that. Re-submit and re-stage.